content/800-data-platform/250-environments/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Environments'
+metaTitle: 'Environments'
+metaDescription: 'In a project in the Prisma Data Platform, you can create multiple environments to associate a Prisma schema on a repository branch with a database and, thus, separate database development activities with JavaScript/TypeScript and Prisma ORM.'
```

Comment:
See my comment above. It feels like this info about JS/TS should be in the intro on the page - but I'm not sure what it means, so I'm not certain of this.